### PR TITLE
mobile-menu

### PR DESCRIPTION
**Pull request by AbbyNyakara**
- Activate the hamburger menu for the mobile view to display the menu  items once its clicked. 
- Each of the links should display the correct part of the page

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const openMenu = document.querySelector('.menu-icon button');
+const closeMenu = document.querySelector('.mobile-menu button');
+
+openMenu.addEventListener('click', () => {
+  document.querySelector('.mobile-menu').classList.add('show');
+});
+
+closeMenu.addEventListener('click', () => {
+  document.querySelector('.mobile-menu').classList.remove('show');
+});
```
Comment (keroloussamy) [inline index.js]: - [x] Kindly as said in the requirements `When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`, when I click on any option the mobile menu disappears Kindly close it.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @AbbyNyakara, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (AbbyNyakara) [reply index.js]: @keroloussamy , I must have missed that. I have added that functionality. Please review again

**Review by keroloussamy — APPROVED** [review]
Hi @AbbyNyakara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**